streaming/src/test/scala/org/apache/spark/streaming/TestSuiteBase.scala

```diff
@@ -557,4 +557,16 @@ trait TestSuiteBase extends SparkFunSuite with BeforeAndAfter with Logging {
       verifyOutput[W](output.toSeq, expectedOutput, useSet)
     }
   }
+
+  /**
+   * Creates a temporary directory, which is then passed to `f` and will be deleted after `f`
+   * returns.
+   * (originally from `SqlTestUtils`.)
+   * @todo Probably this method should be moved to a more general place
+   */
+  protected def withTempDir(f: File => Unit): Unit = {
```

Comment:
I see, it's only otherwise defined in the SQL test utils class. Well something we could unify one day, maybe not such a big deal now here.